modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -926,14 +960,28 @@ public CompletableFuture<Void> stopAsync(ComponentContext componentContext) {
         shutdownAndAwaitTermination(executor, shutdownTimeoutSeconds, TimeUnit.SECONDS);
         shutdownAndAwaitTermination(replicasCreationExecutor, shutdownTimeoutSeconds, TimeUnit.SECONDS);
 
+        try {
+            IgniteUtils.closeAllManually(
+                    () -> CompletableFuture.allOf(raftClientsFutures.values().toArray(new CompletableFuture<?>[0]))
+                            .get(shutdownTimeoutSeconds, TimeUnit.SECONDS),
+                    () -> raftClientsFutures.forEach((raftNodeId, raftClientFuture) -> {
+                        try {
+                            raftManager.stopRaftNode(raftNodeId);
```

Comment:
Am I right that you've wrapped all raftNodes stop into single close?

Reply:
Yes, I see, should be separated